vscode/package.nls.json

```diff
@@ -18,11 +18,11 @@
     "jdk.project.compile": "Compile Project",
     "jdk.project.clean": "Clean Project",
     "jdk.workspace.configureRunSettings": "Edit",
-    "jdk.select.editor.projects": "Reveal active editor in Projects",
+    "jdk.select.editor.projects": "Reveal in Projects",
     "testing.runAll": "Run All Tests",
     "jdk.addEventListener": "Add event listener",
     "jdk.download.jdk": "Download, install and use JDK",
-    "jdk.open.test": "Go To Test/Tested class...",
+    "jdk.open.test": "Go To Test class...",
```

Comment:
This feature does both i.e. You can go to test from a source file and vice versa.
So, can we change title to something which represents above behaviour. Because, updated title seems that it can only go to test class and not vice versa.

Reply:
got it, let me bring back these changes and add appropriate content